raster/src/main/scala/geotrellis/raster/RasterExtent.scala

```diff
@@ -165,4 +168,8 @@ object RasterExtent {
     */
   def apply(extent: Extent, tile: CellGrid[Int]): RasterExtent =
     apply(extent, tile.cols, tile.rows)
+
+  implicit val rasterExtentMergeSemigroup: Semigroup[RasterExtent] = new Semigroup[RasterExtent] {
```

Comment:
The name of the implicit is weird a bit and inconsistent. Here it is `rasterExtentMergeSemigroup` (it can be just `rasterExtentSemigroup`. And for the `Extent` it is `extentIsSemigroup` but it can be `extentSemigroup`.


I'm wondering, do we really need a semigroup here for the `RasterExtent`?